.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and push docker image omnisette-server
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Where is this token from? Is it a personal account, or what?

Reply:
Yeah, my only consideration is that it's going to be with the rights of an individual, even though other orgnaization members will be able to use it through the action. This might lead to impersonation or something... I don't know the correct approach. Some orgs seem to use "bot" accounts for this purpose.

Reply:
So i've changed up the token, and now it uses a token that can only be used during runtime of the action.
This is set here:  https://github.comSideStore/omnisette-server/settings/actions
give Workflow permissions: write permissions
How's this?

Reply:
I can't actually see the settings page you linked because I don't have repo settings permissions... but if it's based on the repository and not just a person, and it works, then that should be good!